platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_structuring_builds.adoc

```diff
@@ -592,3 +592,71 @@ You can also now invoke tasks on the `my-web-module` project using the shorter n
 
 === Tags
 `<<tags_reference.adoc#tag:structuring-builds,#structuring-builds>>`
+
+
+[[use_convention_plugins]]
+== Use Convention Plugins for Common Build Logic
+
```

Comment:
This is so closely associated with "[Favor build-logic Composite Builds for Build Logic](https://docs.gradle.org/release-nightly/userguide/best_practices_general.html#favor_composite_builds)" and `buildSrc`, I wonder if we should mention it in some way. At the very least, this BP needs to come right after the `build-logic` BP.

🤔 It feels odd not to mention `buildSrc` at all.